src/system/data-csv.md

```diff
@@ -22,9 +22,9 @@ A full export of the product database contains information about each product in
 
 The first row of the table contains the names of each attribute, which are used as column headers. The remaining rows describe the individual product records. Any row that begins with a value in the SKU column is the beginning of a new product record. A single product might include several rows that contain information about multiple images or product options. The next row that has a value in the SKU column begins a new product.
 
-The category column contains a path for each category to which the product is assigned. The path includes the root category, followed by a forward slash (/) between each level. By default, the pipe “&#124;” character is used to separate different category paths. For example:
```

Comment:
I'm not sure why documentation says pipe character.  Did it used to be this way?

Reply:
I can't really tell when this information was added. Seems like it has been here for a long time, so I am glad you are correcting it.